src/bp-groups/bp-groups-template.php

```diff
@@ -3660,7 +3661,7 @@ function bp_get_group_member_unban_link( $user_id = 0, $group = false ) {
 	}
 
 	if ( empty( $group ) ) {
-		$group =& $groups_template->group;
+		$group = ! empty( $groups_template->group ) ? $groups_template->group : groups_get_current_group();
```

Comment:
please revert this and whenever it's required please use the argument of the function.